dev-db/postgis/postgis-2.3.2.ebuild

```diff
@@ -128,9 +130,9 @@ src_install() {
 	emake -C topology DESTDIR="${D}" install
 	dobin ./utils/postgis_restore.pl
 
-	dodoc CREDITS TODO loader/README.* doc/*txt
```

Comment:
define `DOCS` here instead (as an array as suggested by @a17r) as the globs won't work otherwise